system/jlib/jsocket.cpp

```diff
@@ -1469,10 +1469,25 @@ void CSocket::setTraceName(const char * prefix, const char * name)
 #ifdef _TRACE
     StringBuffer peer;
     peer.append(prefix);
+    try
+    {
+        int lport;
+        char lname[256];
+        lport = CSocket::name(lname, sizeof(lname));
+        peer.append(":").append(lport);
+    }
+    catch (IException *e)
+    {
+        e->Release();
+    }
+    peer.append(" -> ");
     peer.append(name?name:"(NULL)");
+    peer.append(":").append(hostport);
+    if (sock != INVALID_SOCKET)
+        peer.append(" (").append(sock).append(")");
```

Comment:
Doesn't feel like this should be done here (in setTraceName), but rather by the caller (Can common up elsewhere if same pattern).
e.g., CSocket::CSocket(const SocketEndpoint &ep, ... ,with name where trace name is "T>"
will have target port already and can use it.